paywall/src/utils/postOffice.js

```diff
@@ -87,7 +87,10 @@ export function mainWindowPostOffice(window, iframe, iframeOrigin) {
  * @param {handlerCallback}}handler the callback. This should
  */
 export function setHandler(type, handler) {
-  handlers[type] = handler
+  const currentHandler = handlers[type] || (() => 1)
```

Comment:
hum. Can you please explain what this is doing and the choice of `(() => 1)` for a default?

Reply:
setHandler creates a linked list of callbacks. The `() => 1` case is a null handler that does nothing. It saves 1 character over `() => {}`.

First time you call setHandler, it creates a handler that will call () => 1 and then the handler.
The second time you call sethandler, it creates a handler that calls (() => 1, the first handler), then the 2nd handler.

And so on

Reply:
note to self: use brackets and add no-op comment